src/App.jsx

```diff
@@ -14,70 +16,103 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const ItemGood = (
+  {
+    good, goodsMod, selectedGood, setselectedGood, goodsCh, setGoodsCh,
+  },
+) => {
+  function setNewValues() {
+    const chosen = goodsMod.find(e => e.id === good.id);
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+    chosen.bool = true;
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+    setGoodsCh(goodsMod);
+    setselectedGood(good);
+  }
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+  return (
+    <tr data-cy="Good" className={`${((good.bool && selectedGood !== '') || (good.name === selectedGood.name)) && 'has-background-success-light'}`}>
```

Comment:
Use [classnames](https://www.npmjs.com/package/classnames) library to add classes conditionally
Fix all cases